Design/Colors.swift

```diff
@@ -0,0 +1,230 @@
+//===============================================================
```

Comment:
Is there a reason we need the `Colors.swift` file here too? I noticed it's also at `bin/ColorScript/Resources/Colors.swift`

Reply:
No. I forgot to delete it. Good catch :)